File: CinemaProject.WebAPI/CinemaProject.DAL/Entities/Seat.cs
using System;
using System.Collections.Generic;

namespace CinemaProject.DAL.Entities
{
    public class Seat
    {
        public Guid Id { get; set; }
        public Guid HallId { get; set; }
        public Guid TypeOfSeatId { get; set; }
        public int NumberOfSeat { get; set; }
        public int Row { get; set; }
        public int Column { get; set; }

        public TypeOfSeat TypeOfSeat { get; set; }
        public Hall Hall { get; set; }

        public IList<TicketSeat> SeatReservations { get; set; }
    }
}

File: CinemaProject.Angular/src/app/AdminPages/Models/seat.ts
import { TypeOfSeat } from './typeOfSeat';

class Seat {
  constructor(
    public id: string,
    public numberOfSeat: number,
    public row: number,
    public column: number,
    public hallId: string,
    public typeOfSeatId: string,
    public color: string
  ) {}
}

class SeatView {
  constructor(
    public id: string,
    public numberOfSeat: number,
    public row: number,
    public column: number,
    public hallId: string,
    public typeOfSeatId: string,
    public color: string
  ) {}
}

export { Seat, SeatView };
